dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/utils/AuthUtil.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.dolphinscheduler.api.utils;
+
+import org.apache.dolphinscheduler.common.Constants;
+import org.apache.dolphinscheduler.dao.entity.User;
+
+import javax.servlet.http.HttpServletRequest;
+
+import org.springframework.util.ObjectUtils;
+import org.springframework.web.context.request.RequestContextHolder;
+import org.springframework.web.context.request.ServletRequestAttributes;
+
+public class AuthUtil {
+    
+    private AuthUtil() {
+        throw new IllegalStateException("AuthUtil class");
+    }
+
+    /**
+     * get LoginUser info from request
+     * @return User
+     */
+    public static User user() {
+        ServletRequestAttributes attributes = (ServletRequestAttributes) RequestContextHolder.getRequestAttributes();
```

Comment:
I think a better implementation is to extract user from `LoginHandlerInterceptor` and store in `ThreadLocal`.  `RequestContextHolder` use the same way to store `RequestAttributes`, but adds additional operations to get `SESSION_USER`.
BTW, another advantage of not storing the user in request is that most agents will log the request information, this will expose the user to the log.

Reply:
get SESSION_USER from request used  many places already,   I'll optimize this in another pr